source/uwp/Renderer/lib/AdaptiveUnsupportedElement.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+#include "AdaptiveCards.Rendering.Uwp.h"
+#include "Enums.h"
+#include "UnknownElement.h"
+#include "AdaptiveCardElement.h"
+
+namespace AdaptiveNamespace
+{
+    class DECLSPEC_UUID("e1ab7208-17bc-4fb6-b3c0-c1886987b4cf") AdaptiveUnsupportedElement
```

Comment:
Do we need this class at the platform level? 
They would create their custom class if they need custom parsing and custom object. 

If adaptive card parser is given a json with unsupported type, should we return json with unsupported element in it? I think the answer is YES. O.K it makes sense now. thanks!

If that's the case, shouldn't you add a projection of it in GenerateContainedElementsProjection?

Reply:
see `Util.cpp` :)

---
In reply to: [251094931](https://github.com/Microsoft/AdaptiveCards/pull/2332#discussion_r251094931) [](ancestors = 251094931)

Reply:
Not sure Why my codeflow comment didn't get published... what I meant to say: see changes in `Util.cpp` 👍